elements/pfe-card/src/pfe-card.scss

```
@@ -162,8 +163,16 @@ $size-small: (
   &__header {
     z-index: 2;
     // Declare the header background color
-    background-color: #{pfe-local(BackgroundColor, $region: header)};
-    color:            #{pfe-local(Color, $region: header)};
+    background-color: #{pfe-local($cssvar: BackgroundColor, $region: header)};
+    color:            #{pfe-local($cssvar: Color, $region: header)};
+    :host([on="dark"]) & {
+        // Declare the header background color
+      background-color: #{pfe-local(
+          $cssvar: BackgroundColor--dark,  
+          $region: header
+      )};
+      // $fallback: rgba(255, 255, 255, #{pfe-var(opacity)})
+    }
     @include browser-query(ie11) {
       background-color: #{map-get($pfe-colors, surface--lighter)} !important;
```

```suggestion
      background-color: pfe-fetch(surface--lightest) !important;
```

^ Shoule this still be lighter or should it be lightest?